single-node/calico.yaml

```diff
@@ -0,0 +1,185 @@
+# This ConfigMap is used to configure a self-hosted Calico installation.
```

Comment:
I was thinking the same, but my concern is does it matter that the file would be in the multi-node install?  I'm thinking no?